Code/GraphMol/FilterCatalog/FilterMatchers.cpp

```diff
@@ -63,7 +65,7 @@ SmartsMatcher::SmartsMatcher(const std::string &name, const std::string &smarts,
 SmartsMatcher::SmartsMatcher(const std::string &name, ROMOL_SPTR pattern,
                              unsigned int minCount, unsigned int maxCount)
     : FilterMatcherBase(name),
-      d_pattern(pattern),
+      d_pattern(std::move(pattern)),
```

Comment:
In this case it's okay.  The ROMOL_SPTR pattern is a copy that is passed in so this is preventing two reference count updates.  It is equivalent to const ROMOL_SPTR &pattern apparently.